frontend/class-opengraph-image.php

```diff
@@ -325,6 +360,11 @@ protected function add_image_by_url( $url ) {
 	 * @return void
 	 */
 	protected function add_image_by_id( $attachment_id ) {
+		$is_size_overridden = $this->override_size( $attachment_id );
```

Comment:
Couldn't this line be combined with the next? The variable isn't being reused anyways.